src/UnitfulRecipes.jl

```diff
@@ -1,5 +1,14 @@
 module UnitfulRecipes
 
+@warn """
+UnitfulRecipes has been deprecated.
+
+```
+using Plots, Unitful
```

Comment:
```suggestion
using Unitful, Plots
```

since loading `UnitfulRecipes` in `Plots` is conditioned to have `Unitful` loaded first)

Reply:
Mhh I think the order should not matter, Requires.jl will spawn a job that waits for Unitful to be loaded.

Reply:
TIL !

Reply:
Actually now I am not 100% sure anymore. Requires launches a task in the background, but it might happen, that julia hits `plot(unitful, stuff)`, before the appropriate UnitfulRecipes recipe got added to the method table.
So importing Unitful first, might give the background job a head start.

Reply:
I've tested `using Plots, Unitful`, plot with units, and it works but suggesting `using Unitful, Plots` might be better yes.